develop/plone/i18n/contribute_to_translations.rst

```diff
@@ -87,24 +72,24 @@ You can commit your translation from this locales directory:
 Creating a pull request (no commit access)
 ------------------------------------------
 
-If you do not have commit access on GitHub `collective group <https://github.com/collective>`_.
+If you do not have commit access on GitHub `collective group <https://github.com/collective>`_,
 you can do the following:
 
-Login to GitHub. Go to GitHub `plone.app.locales <https://github.com/collective/plone.app.locales>`_
-
-Press *Fork*. Now GitHub creates a copy of ``plone.app.locales`` package for you.
-
-Then on your computer in ``plone.app.locales`` do a special git push to your own repository::
+- Login to GitHub.
+- Go to GitHub `plone.app.locales <https://github.com/collective/plone.app.locales>`_
+- Press *Fork*.
+  Now GitHub creates a copy of ``plone.app.locales`` package for you.
+- Then on your computer in ``plone.app.locales`` do a special git push to your own repository::
 
     git push git@github.com:YOURUSERNAMEHERE/plone.app.locales.git
 
-Go to GitHub ``https://github.com/YOURUSERNAME/plone.app.locales``
-
-Press button *Create Pull request*. Fill it in.
+- Go to GitHub ``https://github.com/YOURUSERNAME/plone.app.locales``
+- Press button *Create Pull request*.
+  Fill it in.
 
 The request will appear for *plone.app.locales* authors.
-If it does not get merged in timely manner, poke people on the #plone IRC channel
-or the mailing list below (sometimes requests go unnoticed).
+If it does not get merged in timely manner,
+ask on the #plone IRC channel or the mailing list below (sometimes requests go unnoticed).
 
```

Comment:
Still, I would make a decision on that and then do a search and replace on all the docs that point to IRC to point to this overview page. Does it make sense?